example/src/main/java/org/aerogear/mobile/example/ExampleApplication.java

```diff
@@ -3,16 +3,23 @@
 import android.app.Application;
 
 import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.metrics.MetricsService;
+import org.aerogear.mobile.core.metrics.impl.AppMetrics;
+import org.aerogear.mobile.core.metrics.impl.DeviceMetrics;
 
 public class ExampleApplication extends Application {
 
     private MobileCore mobileCore;
+    private MetricsService metricsService;
 
     @Override
     public void onCreate() {
         super.onCreate();
 
         mobileCore = MobileCore.init(this);
+        metricsService = mobileCore.getInstance(MetricsService.class);
+
+        metricsService.sendDefaultMetrics();
```

Comment:
👍  
[Optional] Can we rename this from `Default` to actual name of metrics we send. 
Default sounds vague and since we giving option to call that we should explicity say what metrics will be sent

Reply:
`sendAppAndDeviceMetrics`?

Reply:
Perfect name